cmake/external/libfuzzer.cmake

```diff
@@ -27,6 +27,12 @@ if(TARGET libfuzzer)
   return()
 endif()
 
+# Mark libfuzzer target as done on Windows because fuzzing is not supported.
+if(WIN32)
+  add_custom_target(libfuzzer)
```

Comment:
@wilhuff I just want to confirm that this line indeed marks `libfuzzer` target as done. We do not support fuzzing under Windows.

Reply:
This is the correct way to do it.

You should also short-circuit this way if:
  * The fuzzing option is not set (there's no reason to build the library if we're not fuzzing)
  * OSS Fuzz is supplying its own pre-built fuzzer

This may require plumbing options from the outer build to the inner dependency fetching build.

Reply:
How can we pass the `FUZZING` option when building external projects?

Reply:
By adding to the `CMAKE_ARGS` you pass to `ExternalProject_Add`.

In this case, add `-DFUZZING=${FUZZING}` to the CMake invocation here: https://github.com/firebase/firebase-ios-sdk/blob/master/cmake/external_rules.cmake#L19

Reply:
Done, thanks.